src/components/Palmid/hooks/useFasta.tsx

```diff
@@ -33,7 +33,7 @@ export function useFasta(): FastaHook {
     }
 
     useEffect(() => {
-        if (checkReportRequestCount >= 60) {
+        if (checkReportRequestCount >= 120) {
```

Comment:
@ababaian's comment helped me understand this a bit better. From the context of this file alone, it's not clear what length of time `120` translates to. I'd recommend adding a comment to make this more understandable:

```suggestion
        // This translates to 120 * REQUEST_INTERVAL = 10 minutes.
        // This should represent the maximum execution time of the Lambda function.
        if (checkReportRequestCount >= 120) {
```

Reply:
@hannahle I suggested `REQUEST_INTERVAL` because otherwise it's unclear where the 5 seconds is coming from. Mentioning it here would also make it more likely that this comment is updated if `REQUEST_INTERVAL` is ever changed in the future. Do you think a similar hint could be added?